avail/src/main/kotlin/avail/compiler/ModuleHeader.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * ModuleHeader.kt
+ * HeaderModule.kt
```

Comment:
This was changed and now disagrees with the file name.  Same for line 71, which should use `[]` to ensure the names agree after renames.